balboa-agent/src/main/scala/com/socrata/balboa/agent/BalboaAgent.scala

```diff
@@ -52,15 +40,21 @@ object BalboaAgent extends App with Config with LazyLogging {
   logger info s"Starting Balboa Agent.  Consuming metrics from ${dataDir.getAbsolutePath}.  " +
     s"AMQ Server: $activemqServer, AMQ Queue: $activemqQueue $additionalInfo".trim
 
-  val mc = (activemqUser, activemqPassword) match {
-    case (Some(user), Some(password)) => new MetricConsumer(
-      dataDir, MetricJmsQueue.getInstance(user, password, activemqServer, activemqQueue))
-    case _ => new MetricConsumer(dataDir, MetricJmsQueue.getInstance(activemqServer, activemqQueue))
+  val amqConnection = (activemqUser, activemqPassword) match {
+    case (Some(u), Some(p)) => ActiveMQConnection.makeConnection(u, p, activemqServer)
```

Comment:
Wish you wouldn't do this. 'user' and 'password' are not long variable names, and it makes it really easy to read this line.